src/app/lib/analyticsUtils/index.js

```diff
@@ -203,3 +194,13 @@ export const getClickInfo = (elem, { service, component, label, type }) => {
 
   return `PUB-[${service}-${component}]-[=${type}]-[${label}]-[${format}]-[]-[]-[]-[${url}]`;
 };
+
+export const getLibraryVersion = platform => {
+  if (platform === 'amp') {
```

Comment:
I think we can refactor this to be...
```suggestion
  if (platform === 'amp' || onClient()) {
    return 'simorgh';
  }
  return 'simorgh-nojs';
```